src/com/facebook/buck/apple/AppleBundle.java

```diff
@@ -409,7 +421,7 @@ public boolean isLegacyWatchApp() {
     }
 
     if (!frameworks.isEmpty()) {
-      Path frameworksDestinationPath = bundleRoot.resolve(this.destinations.getFrameworksPath());
```

Comment:
What do you mean by "creating a step inside `AppleBundle`"? Refactor the existing "copy Swift stdlibs" logic that was already there to support both copying them to `/Frameworks` as well as the enclosing `.ipa` (when called from `BuiltinApplePackage`?

My comment below was referring to the changes to `ApplePackageDescription` (as `AppleBundle` already has a reference to the tooling).